src/CMakeLists.txt

```diff
@@ -818,13 +818,43 @@ add_definitions(-DFREERTOS)
 add_definitions(-D__STACK_SIZE=1024)
 add_definitions(-D__HEAP_SIZE=4096)
 
-# NOTE : Add the following defines to enable debug mode of the NRF SDK:
-#add_definitions(-DDEBUG)
-#add_definitions(-DDEBUG_NRF_USER)
-
-if (NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE "Release")
-endif ()
+# Target hardware configuration options
+add_definitions(-DTARGET_DEVICE_${TARGET_DEVICE})
+if(LF_CLK STREQUAL "RC")
+        add_definitions(-DCLOCK_CONFIG_LF_SRC=0)
```

Comment:
in the top level CMakeLists.txt we have 2 spaces per tab, this file has sometimes 4 spaces per tab and for file indentations 8 spaces per tab. The C++ source has 2 spaces per tab. The doc/coding-convention.md file also specifies 2 spaces for indentation.

I think it would be a good idea to use 2 spaces per tab for new/changed code

Reply:
Okay, I adjusted the file to 2 spaces per indentation.